framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1879,6 +1879,8 @@ public AnnotatedDeclaredType getSelfType(Tree tree) {
         if (enclosingClass.getSimpleName().length() != 0 && enclosingMethod != null) {
             AnnotatedDeclaredType methodReceiver;
             if (TreeUtils.isConstructor(enclosingMethod)) {
+                // This seems questionable, since it's the type of `this` after the constructor
```

Comment:
This seems like a `TODO` comment. Is this a concern for how FbC is implemented? Or also for other type systems?

Reply:
It's for all initialization type systems (rawness too).  But probably not for anything else, because other type systems don't change the type of `this` throughout the constructor.

Yes, it is a TODO comment.  I was hoping you could double-check whether my concern is valid.

Reply:
(I didn't edit to add "TODO" because that would resolve this conversation.)